app/controllers/submissions_controller.rb

```diff
@@ -225,7 +225,6 @@ def collect_submissions
                      assignment_identifier: assignment.short_identifier)
       flash_now(:error, error)
     end
-    render 'shared/_poll_job'
```

Comment:
This is good, but in general prefer an explicit `head :success` to send just an HTTP status rather than use whatever gets rendered by default.